docs/src/pages/getting-started/templates/pricing/Pricing.tsx

```diff
@@ -139,62 +92,54 @@ const footers = [
   },
 ];
 
-const defaultTheme = createTheme();
-
 function PricingContent() {
-  const classes = useStyles();
-
   return (
     <React.Fragment>
+      <GlobalStyles styles={{ ul: { margin: 0, padding: 0, listStyle: 'none' } }} />
       <CssBaseline />
       <AppBar
         position="static"
         color="default"
         elevation={0}
-        className={classes.appBar}
+        sx={{ borderBottom: (theme) => `1px solid ${theme.palette.divider}` }}
       >
-        <Toolbar className={classes.toolbar}>
-          <Typography
-            variant="h6"
-            color="inherit"
-            noWrap
-            className={classes.toolbarTitle}
-          >
+        <Toolbar sx={{ flexWrap: 'wrap' }}>
+          <Typography variant="h6" color="inherit" noWrap sx={{ flexGrow: 1 }}>
             Company name
           </Typography>
           <nav>
             <Link
               variant="button"
               color="text.primary"
               href="#"
-              className={classes.link}
+              sx={{ my: 1, mx: 1.5 }}
             >
               Features
             </Link>
             <Link
               variant="button"
               color="text.primary"
               href="#"
-              className={classes.link}
+              sx={{ my: 1, mx: 1.5 }}
             >
               Enterprise
             </Link>
             <Link
               variant="button"
               color="text.primary"
               href="#"
-              className={classes.link}
+              sx={{ my: 1, mx: 1.5 }}
             >
               Support
             </Link>
           </nav>
-          <Button href="#" variant="outlined" className={classes.link}>
+          <Button href="#" variant="outlined" sx={{ my: 1, mx: 1.5 }}>
             Login
           </Button>
         </Toolbar>
       </AppBar>
       {/* Hero unit */}
-      <Container maxWidth="sm" component="main" className={classes.heroContent}>
+      <Container maxWidth="sm" component="main" sx={{ pt: 8, pb: 6, px: [0] }}>
```

Comment:
?

```suggestion
      <Container maxWidth="sm" component="main" sx={{ pt: 8, pb: 6, px: 0 }}>
```

Reply:
I had to add `[0]` since placing only `0` does not overwrite the style of the media query. Same for this https://github.com/mui-org/material-ui/pull/26604#discussion_r646002903

Reply:
Intuitively, I would say that
```suggestion
      <Container maxWidth="sm" component="main" sx={{ pt: 8, pb: 6 }} disableGutters>
```

might be cleaner, but now, looking at the visual output, this seems much better:

```suggestion
      <Container maxWidth="sm" component="main" sx={{ pt: 8, pb: 6 }}>
```

Reply:
Done